front/lib/resources/file_resource.ts

```diff
@@ -303,11 +303,11 @@ export class FileResource extends BaseResource<FileModel> {
     const updateResult = await this.update({ status: "ready" });
 
     // For Content Creation conversation files, automatically create a ShareableFileModel with
-    // default conversation_participants scope.
+    // default none scope.
```

Comment:
Actually does this make sense? I feel like this model (sharedBy, sharedAt) is not really true when it's `none` (no longer sharable)?

Reply:
would it be okay if I rename `sharedBy` & `sharedAt` to editedBy and `editedAt`? 🤔

Reply:
Agree with the comment above better to keep sharedBy and sharedAt at null if scope is none and not rename to editBy/editAt